### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by mcmenaminadrian**
Some start up text in the best traditions of Spectrum crapiness I believe

### CODE REVIEW

File: SplitFiles/game.c [hunk 1]
```diff
@@ -73,6 +73,13 @@ int main_game()
     int i;
     //Clear screen
     zx_cls(PAPER_WHITE);
+    printf(PRINTAT "\x01\x01" \
+      "Welcome to the " "\x10\x32H\x10\x34o\x10\x36t\x10\x35" "e\x10\x31l " "\x10\x32" "E" \
+      "\x10\x34" "x" "\x10\x36" "c" "\x10\x35" "e" "\x10\x31" "s\x10\x33s\x10\x30.");
+    printf(PRINTAT "\x01\x03" "We loved your job application to");
+    printf(PRINTAT "\x01\x04" "be our new breakfast Toast Host.");
+    printf(PRINTAT "\x01\x05" "But now stuff gets really real.");
+    printf(PRINTAT "\x01\x07" "\x12\x31\x10\x32" "Can you handle the pressure?\x12\x30\x10\x30");
```
Comment (ConorMcPersonal) [inline SplitFiles/game.c:82]: Love it - feels like I am playing Football Manager (http://torinak.com/qaop/play/manager)

File: SplitFiles/control.c [hunk 2]
```diff
@@ -114,7 +113,20 @@ void execute_command(ControlBuffer *ctrlBuff, GameParameters* params) {
             params->messageAddress = 100 + d - '0';
             BreadState* new_slice = malloc(sizeof(struct BreadStateStruct));
             new_slice->temperature = 0;
-            new_slice->moisture = ((e == 'B')? 75: 50) + rand()%50; //Brown takes longer
+            switch(e) {
```
Comment (ConorMcPersonal) [inline SplitFiles/control.c:116]: This is a great start - but long term I saw the bread types as being data-driven so they can be randomly ordered by customers and have different characteristics and scoring profile (ie harder to toast stuff gets ordered less often but scores more).

**Comment by mcmenaminadrian** [comment]
That makes total sense - need to generate an order list?

**Comment by ConorMcPersonal** [comment]
At some point we should add random order creation, yes.  In the meantime go ahead and merge this.

**Comment by ConorMcPersonal** [comment]
Ah sorry - think I f*cked it again.  I don't use PRs at work so not too good with them.  Please merge this to the main branch!!

**Pull request merged**